added router

keiryan opened the pull request:
Fixed the incorrect routing that would not switch pages.

keiryan committed:
added router

yoloOnTheBattlefield commented on src/App.js:
```diff
@@ -10,13 +10,13 @@ import {
 import Navbar from "./components/Navbar/navbar";
 import { NavBarNotch } from "components/NavBarNotch/navbarnotch";
-import CoinContainer from "./components/CoinContainer/coincontainer";
+// import CoinContainer from "./components/CoinContainer/coincontainer";
```
no commented out code

yoloOnTheBattlefield commented on src/utils/MarketCap/index.js:
```diff
@@ -6,7 +6,6 @@ export function getMarketCap(value){
     //     2: '',
     // }
```
no commented out code

yoloOnTheBattlefield reviewed (commented):
https://developer-pro.mykajabi.com/products/react-pro/categories/2149030003/posts/2152230042

keiryan committed:
fixed various features, and added improvements

keiryan commented:
### Pull Request 2 Change log:

- Implemented coin API
- Updated absolute imports in App.js
- Added bold coin name styling
- Fixed number abbreviator algorithm
- Coin percentages have been moved to`<abbr>` tags to reduce space taken overall. The full percentage can be seen by hovering over the desired percentage.
- Barchart numbers have also been moved to`<abbr>` tags if full number is desired to be seen.
- Scroll bar has been set to a custom color to reduce intrusion.
- Removed all `console.log()`'s
- Removed unnecessary commented out code

yoloOnTheBattlefield commented on src/components/Navbar/navbar.styles.js:
```diff
@@ -1,6 +1,6 @@
-import {Base} from '../../app.styles'
-import {Link} from 'react-router-dom'
+import { Base } from '../../app.styles'
+import { Link } from 'react-router-dom'
```
absolute before relative

yoloOnTheBattlefield commented on src/pages/Coinpage/coinpage.js:
```diff
@@ -1,11 +1,24 @@
 import React from "react";
+import CoinContainer from '../../components/CoinContainer/coincontainer'
+import { MiddleContainer } from './coinpage.styles'
+import axios from 'axios';
```
absolute before relative

keiryan committed:
fixed the absolute imports
